.github/workflows/test.yml

```diff
@@ -47,5 +47,7 @@ jobs:
         ln -s $(which python) /usr/bin/python3
 
     - run: python ./get-toolchain.py
+      env:
+        GH_TOKEN: ${{ github.token }}
```

Comment:
Does this need to be marked as a "secure" environment variable or something?

Reply:
No. It is secure, by itself. That is, the content of `github.token` depends on the context. If it is executed after someone with write privileges pushed a commit, it has some permissions. If it is executed in a fork, it has the permissions of the forker. If it is executed in a PR, it has read permissions but not write permissions.

This is not a Personal Access Token, which you would need to add to the "Secrets" in the settings of the repo, but it is a secret.

Reply:
See https://docs.github.com/en/free-pro-team@latest/actions/reference/authentication-in-a-workflow#permissions-for-the-github_token